clink/app/src/loader/clink.bat

```diff
@@ -5,6 +5,19 @@
 set clink_profile_arg=
 set clink_quiet_arg=
 
+set clink_autorun=0
+set clink_parameters="%*"
```

Comment:
This line hits a syntax error with input such as:
```bat
clink inject --profile "c:\somewhere\abc & xyz"
```
That is a valid directory name to exist.

Reply:
That's no good either:  the usage of `if defined` relies on Command Extensions, and causes `clink.bat` to fail whenever Command Extensions is disabled.  And the combination of autorun plus having Command Extensions disabled will cause every cmd invocation to report errors, interfering with scripted usage in potentially catastrophic ways.

I think I see a way to address that, as well.

Reply:
Urg.  `clink.bat` uses `if /i` which relies on command extensions.

So, this investigation has also revealed that `clink.bat` explodes when command extensions are disabled.

I'll mitigate that by adding `setlocal enableextensions` at the top, to force command extensions to be enabled.  That won't work on very (very!!) old versions of CMD, but Clink itself doesn't work on such old versions of CMD anyway, so that's ok.